python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -275,6 +275,24 @@ def __iter__(self):
         for i, name in enumerate(group_names):
             yield name, grouped_values[offsets[i] : offsets[i + 1]]
 
+    @property
+    def dtypes(self):
+        non_grouped = [
+            name
+            for name in self.obj._data.names
+            if name not in self.grouping.names
+        ]
+        all_dtypes = self.obj._dtypes
+        index = self.grouping.keys.unique().to_pandas()
+        df = pd.DataFrame(
+            {
+                name: pd.Series([all_dtypes[name]]).repeat(len(index))
+                for name in non_grouped
+            }
+        )
+        df.index = index
+        return df
```

Comment:
This looks much more simple than the pandas implementation, but I guess it makes sense.